iogt/templates/base.html

```diff
@@ -80,6 +80,11 @@
 </main>
 
 {# Global javascript #}
+<script src="{% url 'javascript-catalog' %}"></script>
+<script>
+    // for JS enabled devices hide double menu
+    document.querySelector('.footer-head').style.display = "none";
+</script>
```

Comment:
Let's avoid inline JS if possible. Please move this back into 'iogt.js' and 'iogt-no-jquery.js'.